keyboards/knops/mini/readme.md

```diff
@@ -0,0 +1,36 @@
+knops keyboard firmware
```

Comment:
Would you mind updating this file to be consistent with [our template](https://docs.qmk.fm/documentation_templates.html#keyboard-readmemd-template)?

Reply:
sure! I used the one generated by the add project shell script, but will modify it.